5-causation/01-defining-causality/notes.qmd

```diff
@@ -283,17 +283,9 @@ df %>%
   )
 ```
 
-It is easier to come up with plausible-seeming estimates
-of average treatment effects than individual treatment effects
-because we see subjects under both treatment and control conditions.
-But to decide if estimates like the one we've just given are reliable
-we need to think about how the data for the two groups was generated and
-whether they serve as good approximations to each other's counterfactuals.
-This question will be the focus of the rest of the causality unit.
-
-
 :::
 
+It is easier to come up with plausible-seeming estimates of average treatment effects than individual treatment effects because we see subjects under both treatment and control conditions. But to decide if estimates like the one we've just given are reliable we need to think about how the data for the two groups was generated and whether they serve as good approximations to each other's counterfactuals. This question will be the focus of the rest of the causality unit.
```

Comment:
I also fixed the paragraph formatting here (better to not do manual line breaks on paragraphs)